src/renderer/scss/RecentConnections.scss

```diff
@@ -53,3 +53,31 @@ td {
 .deleteButton:hover {
   background-color: rgb(247, 68, 68);
 }
+
+.edit-button {
+  background-color: inherit;
+  border: 5px;
+  border-radius: 5px;
+}
+
+.edit-button:hover {
+  background-color: lightgreen;
+}
+
+.edit-submit-btn {
+  margin: 10px 0px;
+}
+
+.edit-button {
```

Comment:
Not sure how these classes got in there twice - let's remove the duplicates

Reply:
Removed